presto-raptor/src/main/java/com/facebook/presto/raptor/storage/organization/OrganizationJob.java

```diff
@@ -44,13 +45,15 @@
     private final ShardManager shardManager;
     private final ShardCompactor compactor;
     private final OrganizationSet organizationSet;
+    private int priority;
```

Comment:
I don't think `OrganizationJob` need to copy the priority out to its private member. Can it just refer to `organizationSet.getPriority()`?